src/api/spec/controllers/webui/cloud/azure/upload_jobs_controller_spec.rb

```diff
@@ -0,0 +1,230 @@
+require 'rails_helper'
+require 'webmock/rspec'
+
+RSpec.describe Webui::Cloud::Azure::UploadJobsController, type: :controller, vcr: true do
+  let(:azure_configuration) { build(:azure_configuration, application_id: 'Hey OBS!', application_key: 'Hey OBS?') }
+  let(:user_with_azure_configuration) { create(:confirmed_user, login: 'tom', azure_configuration: azure_configuration) }
+  let(:project) { create(:project, name: 'Apache') }
+  let!(:package) { create(:package, name: 'apache2', project: project) }
```

Comment:
:thought_balloon: is there a reason you call the project/package that way? I find this a bit confusing with the old test suite...

Reply:
It is because I copied what already was there (from Chris PR). Will change both

Reply:
Because you need to pass in the project name.
What is wrong with it?